src/templates/app/cashier/account_transfer.jsx

```diff
@@ -12,72 +12,64 @@ const Row = ({ id }) => (
+                </span>
+                <span className='invisible' id='no_balance'>
+                    {it.L('Please [_1]deposit[_2] to your account.', `<a href='${it.url_for('cashier/forwardws?action=deposit')}'>`, '</a>')}
+                    &nbsp;
+                </span>
+                <span className='invisible' id='limit_reached'>{it.L('You have reached your withdrawal limit.')}&nbsp;</span>
+            </p>
+        </div>
 
-            <div className='invisible' id='success_form'>
-                <p>{it.L('Your fund transfer is successful. Your new balances are:')}</p>
-                <Row id='from' />
-                <Row id='to' />
-                <p>
-                    <a href='javascript:;' id='reset_transfer'>{it.L('Make another transfer')}</a>
-                </p>
-            </div>
+        <div className='invisible' id='error_message'>
+            <p className='center-text notice-msg' />
+        </div>
 
-            <form className='invisible' id='frm_account_transfer'>
-                <p>{it.L('Transfer funds between your real money accounts.')}</p>
+        <div className='invisible' id='success_form'>
+            <p>{it.L('Your fund transfer is successful. Your new balances are:')}</p>
+            <Row id='from' />
+            <Row id='to' />
+            <p>
+                <a href='javascript:;' id='reset_transfer'>{it.L('Make another transfer')}</a>
+            </p>
+        </div>
 
-                <Fieldset legend={it.L('Details')}>
-                    <FormRow label={it.L('Transfer from')} type='label'  id='lbl_transfer_from' />
-                    <FormRow label={it.L('Transfer to')}   type='select' id='transfer_to' />
-                    <FormRow label={it.L('Amount')}        type='custom' id='transfer_amount'>
-                        <label id='currency' />
-                        <input id='amount' name='amount' type='text' maxLength='20' autoComplete='off' />
-                        <div className='hint' id='range_hint' />
-                    </FormRow>
-                </Fieldset>
+        <form className='invisible' id='frm_account_transfer'>
+            <p>{it.L('Transfer funds between your real money accounts.')}</p>
 
-                <SubmitButton msg_id='form_error' type='submit' text={it.L('Transfer')} />
-            </form>
+            <Fieldset legend={it.L('Details')}>
+                <FormRow label={it.L('Transfer from')} type='label'  id='lbl_transfer_from' />
+                <FormRow label={it.L('Transfer to')}   type='select' id='transfer_to' />
+                <FormRow label={it.L('Amount')}        type='custom' id='transfer_amount'>
+                    <label id='currency' />
+                    <input id='amount' name='amount' type='text' maxLength='20' autoComplete='off' />
+                    <div className='hint' id='range_hint' />
+                </FormRow>
+            </Fieldset>
 
-            <div className='hint invisible' id='transfer_info'>
```

Comment:
we can't remove old code, this needs to become visible in some cases, and the second longer list needs to become visible in other cases. you can have a look at the corresponding JS file and debug why in your branch it's showing both at the same time